### ISSUE TITLE
Example plugin vr input suddenly broken across all installs of the example project

### ISSUE THREAD

**Issue by give-me-pen** [mordentral/VRExpPluginExample#32]
I'm relatively new to Unreal- migrated from Unity, so bear with me. Also never interacted with github, so apologies for poor etiquette 

I was trying to test out some of the features of the demo with some mechanics I intended to build in my own project (pretty much just some animation IK), and for a while everything worked perfectly. Then somehow, for whatever reason, everything broke. Tracking half works, but the rotation isn't correct, and on top of that, none of the inputs work. 

I wouldn't be asking for help here ,if not for whatever reason, the problem still persisting despite re-downloading the project and launching everything with default settings. So quite frankly, I have no idea where to even start with fixing the issue

I both restarted steamvr and checked that it was still running properly in other games, and it worked fine. Still freaking out in the example projects however

I'm using Unreal version 5.3.2, which I doubt the editor version is a problem since it was working fine on the same version, but it could be for all I know

**Comment by give-me-pen** [comment]
Okay, so even after I had reinstalled steam vr and tried it on an engine default vr template, no tracking. tracking still worked in games however, such as beat saber. So I tried it in unity editor and it was still broken. As a last stitch attempt I restarted steamvr one more time and all of a sudden everything works.
No idea what went wrong or how to recreate it, no idea why it suddenly fixed itself after the final restart, no idea how I still had some tracking with this plugin, but I do know it was definitely a Steam VR issue
